server/src/main/java/org/candlepin/dto/manifest/v1/CertificateSerialDTO.java

```diff
@@ -36,6 +36,9 @@ public class CertificateSerialDTO extends TimestampedCandlepinDTO<CertificateSer
     protected BigInteger serial;
     protected Date expiration;
     protected Boolean revoked;
+    protected Boolean collected = Boolean.FALSE; // we export this to false by default now that the CRL
```

Comment:
Does this comment block pass the linter? Generally we put these as a block above the target line.

Reply:
it does pass. what target line?

Reply:
ah I see what you mean

Reply:
updated